components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/helpers.py

```diff
@@ -24,7 +24,7 @@ def get_prefixed_index_html():
     with open(os.path.join(static_dir, "index.html"), "r") as f:
         index_html = f.read()
         index_prefixed = re.sub(
-            r"\<base href=\".*\"\>", '<base href="%s">' % prefix, index_html,
+            r"\<base href=\".*\".*\>", '<base href="%s">' % prefix, index_html,
```

Comment:
What's the issue we are trying to fix here? Do we need to match **anything** that follows the `href` in the `<base>` tag?

Reply:
@elikatsis I wanted to catch the use case where the `base` tag could look like this, which includes a space as well
```html
<base href="/" >
```

Reply:
Then I'd probably suggest using a regex like `r"\<base href=\".*\"\s?\>"` just to make things a bit more strict. Would this work?

Reply:
Yeap, I'll use that instead

Reply:
@elikatsis in the end it's not just the white spaces. We have two cases:
1. `<base href="/tensorboards/">`
2. `<base href="/" />`

There's also an extra `/` character in the end that I missed. So if you are OK I'll stick to the current catch-all rule I had which captures this as well, since we will be complicating the regex more to handle the `/`.

Reply:
Sure let's move on with this as is and come back to it in the future if there's any reason to do so.